src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -228,25 +230,32 @@ private void configureBindings() {
                 RobotConstants.ROTATION_FF));
 
     m_manipController.y().whileTrue(m_Elevator.moveElevatorUp());
-    m_manipController.x().whileTrue(m_Elevator.moveElevatorDown());
+    //  m_manipController.x().whileTrue(m_Elevator.moveElevatorDown());
 
     m_manipController
         .leftTrigger(0.5)
         .whileTrue(m_Shooter.startShooter(500))
         .whileFalse(m_Shooter.stopCommand());
 
-    m_manipController.a().whileTrue(m_Wrist.moveWristUp());
+    // m_manipController.a().whileTrue(m_Wrist.moveWristUp());
+    // Manual wrist Movement
+    // m_manipController.b().whileTrue(m_Wrist.moveWristDown());
 
-    m_manipController.b().whileTrue(m_Wrist.moveWristDown());
+    m_testController.a().whileTrue(m_Wrist.setToTarget(90)).whileFalse(m_Wrist.setToTarget(130));
+
+    m_manipController
+        .y()
+        .whileTrue(m_Wrist.setToTarget(110).alongWith(m_Elevator.setToTarget(13.9))) // Sets to AMP
+        .whileFalse(m_Wrist.setToTarget(139).alongWith(m_Elevator.setToTarget(0))); // sets to STOW
+
+    m_manipController.x().whileTrue(m_Wrist.setToTarget(125)).whileFalse(m_Wrist.setToTarget(139));
```

Comment:
Make a default command in `Wrist` that sets it to `139` when no other command is commanding that subsystem.